CONTRIBUTING.md

```diff
@@ -1,6 +1,27 @@
 For contributing to Spoon, the recommended way is to create a pull request (PR) on Github. Upon acceptance, the merged code is copyrighted under the [Cecill-C license](http://www.cecill.info/licences/Licence_CeCILL-C_V1-en.html).
 
-Guidelines for pull requests:
+The integrator code of conduct
+------------------------------
+
+The integrators are the developers who have write access to the repository. In particular, they can merge pull requests to the `master` branch. They shall respect the following rules:
+
+* One never pushes directly to `master`
+* One only merges atomic pull requests (single bug fix or single feature)
+* One only merges well tested and well documented pull requests, after thorough code review
+* One never merges one's own pull request
+* One leaves pull requests opened at least 1 day before merging, so that the community is aware and can comment on them
```

Comment:
add a link to the commit message convention: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#type